Formula/v/vedic.rb

```diff
@@ -0,0 +1,22 @@
+class Vedic < Formula
+  desc "Simple Sanskrit programming language"
+  homepage "https://vedic-lang.github.io/"
+  url "https://github.com/vedic-lang/vedic/archive/refs/tags/v2.0.6.tar.gz"
+  sha256 "5453386fcebfa48a5f3d39bc832d19edaaaa98d8d36841428ff6a20db0dd0151"
```

Comment:
sha got changed due to re-tagged.
```suggestion
  sha256 "9bfb68dfa8a79c02d52905eb1403267209dae80ad05287b7f3706f14071c4800"
```